CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to Tangram ES
+
+The easiest way to help out is to submit bug reports and feature requests on our [issues](https://github.com/tangrams/tangram-es/issues) page.
+
+When submitting a bug report, please include:
+
+ - The device and operating system version that produced the bug
+ - Steps required to recreate the issue
+ - What happened
+ - What you expected to happen
```

Comment:
Maybe add another points of what version of tangram-es was being used? A released version or master.